src/mpi/topo/dims_create.c

```diff
@@ -410,9 +410,6 @@ static void factor_to_dims_by_rr(int nf, Factors f[], int nd, int dims[])
    values are known.  Then pass in the entire array.  This is needed
    to get the correct values for "ties" between the first and last values.
  */
-#undef FC_NAME
```

Comment:
nitpick: this part could be separated out from the functional changes

Reply:
Thanks. Split and updated.